auth-web/nginx.conf

```diff
@@ -66,6 +62,7 @@ http {
     }
 
      location @index {
+      add_header Cache-Control no-cache;
```

Comment:
Never want index to cache, because it contains all the hashes for the new files.